vertexai/src/VertexAI.cs

```diff
@@ -106,6 +106,32 @@ public GenerativeModel GetGenerativeModel(
         generationConfig, safetySettings, tools,
         toolConfig, systemInstruction, requestOptions);
   }
+
+  /// <summary>
+  /// Initializes a `LiveGenerativeModel` for real-time interaction.
+  /// 
+  /// - Note: Refer to [Gemini models](https://firebase.google.com/docs/vertex-ai/gemini-models) for
+  /// guidance on choosing an appropriate model for your use case.
+  /// </summary>
+  /// <param name="modelName">The name of the model to use, for example `"gemini-2.0-flash-live-preview-04-09"`; see
```

Comment:
we should have a more stable live model name now.

Reply:
Looking around, I couldn't find a great one. I'll switch it to "...flash-exp" for now, since that seems a bit more generic then the very long one with the date, and so might be a bit more stable.

Reply:
After discussing it more, we will keep the longer name, since that is what the other docs have for now.